src/Command/SqlGeneratorCommand.php

```diff
@@ -0,0 +1,229 @@
+        $longestDataTypeLength = $this->getLongestByType($data, 'dataType');
+        $toReturn = '';
+
+        foreach ($data as $row) {
+            $nameLength = \mb_strlen($row['name']);
+            $dataTypeLength = \mb_strlen($row['dataType']);
+            $toReturn .= self::INDENTATION . $row['name'] . \str_repeat(' ', $longestNameLength - $nameLength + 1);
+            $toReturn .= $this->buildDataType($row, $longestDataTypeLength, $dataTypeLength);
+            $toReturn .= $row['notNull'] === true ? ' NOT NULL' : '';
+            $toReturn .= $row['default'];
+            $toReturn .= $this->buildAutoIncrement($row['name']);
+            $toReturn .= ',' . \PHP_EOL;
+        }
+
+        return $toReturn;
+    }
+
+    private function buildAutoIncrement(string $name) : string
+    {
+        return $name === '`id`' ? ' AUTOINCREMENT' : '';
+    }
+
+    private function buildDataType(array $row, int $longestDataTypeLenght, int $dataTypeLenght) : string
+    {
+        if ($row['notNull'] === false && $row['default'] === '') {
+            return $row['dataType'];
+        }
+
+        if ($row['notNull'] === false && $row['default'] !== '') {
+            return $row['dataType'] . \str_repeat(' ', $longestDataTypeLenght - $dataTypeLenght + 9);
+        }
+
+        return $row['dataType'] . \str_repeat(' ', $longestDataTypeLenght - $dataTypeLenght);
+    }
+
+    private function getLongestByType(array $data, string $type) : int
+    {
+        $maxLength = 0;
+
+        foreach ($data as $row) {
+            $length = \mb_strlen($row[$type]);
+
+            if ($length > $maxLength) {
+                $maxLength = $length;
+            }
+        }
+
+        return $maxLength;
+    }
+
+    private function getData(\ReflectionProperty $property) : array
+    {
+        return [
+            'name' => $this->getPropertyName($property),
+            'dataType' => $this->getDataType($property),
+            'notNull' => $this->isNotNull($property),
+            'default' => $this->getDefault($property),
+        ];
+    }
+
+    private function getDefault(\ReflectionProperty $property) : string
+    {
+        $type = $property->getType();
+        $defaultValueAttribute = $property->getAttributes(\CoolBeans\Attribute\FunctionDefaultValue::class);
+
+        if (!$property->hasDefaultValue() && \count($defaultValueAttribute) === 0) {
+            return '';
+        }
+
+        if (\count($defaultValueAttribute) === 1) {
+            return ' DEFAULT ' . $defaultValueAttribute[0]->getArguments()[0];
+        }
+
+        $defaultValue = $property->getDefaultValue();
+
+        if ($defaultValue === null) {
+            return ' DEFAULT NULL';
+        }
+
+        if ($type->getName() === 'bool') {
+            return ' DEFAULT ' . ($defaultValue === true ? '1' : '0');
+        }
+
+        return ' DEFAULT "' . $defaultValue . '"';
+    }
+
+    private function isNotNull(\ReflectionProperty $property) : bool
```

Comment:
This could also follow the same mechanic as other functions and return "NOT NULL" or "        " depending on the type - getNotNull()